src/app/app.component.ts

```diff
@@ -9,28 +10,35 @@ import 'rxjs/add/operator/mergeMap';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss'],
+  styleUrls: ['./app.component.scss']
 })
 
 
 
 export class AppComponent implements OnInit {
+  private scrolledState = false;
```

Comment:
Any specific reason to maintain state for scroll @live-wire ?

Reply:
I wanted the header elements to change color when the page is scrolled.